app/customer.py

```diff
@@ -0,0 +1,57 @@
+import math
+from datetime import datetime
```

Comment:
Import this as just datetime
```suggestion
import datetime
```